market/demos/master-detail/meta.json

```diff
@@ -13,7 +13,7 @@
 		{
 			"name": "Master Detail",
 			"groupId": "com.axonivy.demo.masterdetail",
-			"artifactId": "master-detail-demo",
+			"artifactId": "master-detail-product",
```

Comment:
didn't see this at first @nqhoan-axonivy  ... but we must always refer to the 'product' artifact. otherwise the market will just show a blank page for this product.

Reply:
oh, got it.
thank @ivy-rew for correcting my bad!

Reply:
no problem, should have seen the mistake too ;-) ... we all learn on the way :+1: